doc/api/deprecations.md

```diff
@@ -40,7 +40,7 @@ However, the deprecation identifier will not be modified.
 
 ## List of deprecated APIs
 
-<a id="DEP0001"></a>
+<i id="DEP0001"></i>
```

Comment:
```suggestion
<wbr id="DEP0001"/>
```

@aduh95, Beats `<hr/>` because there is no CSS involved. This is super clean. WDYT?

Refs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr

Reply:
If @Trott were to reconsider his stance and simply allowed us to sanitize all HTML tags in the headings we wouldn't wind up w/ this suboptimal hack. Can we please do this as it is normally done? Resorting to having to re-appropriate tags feels like we're going way out of our way.

Please think about it.

Reply:
> If @Trott were to reconsider his stance and simply allowed us to sanitize all HTML tags in the headings

Sure I can remove the request-changes. 

I'd prefer the markdown headings not have HTML tags within them, but if we need to do that, then oh well, sure.